easybuild/easyconfigs/e/Embree/Embree-3.13.2-foss-2022b.eb

```diff
@@ -0,0 +1,39 @@
+name = 'Embree'
+easyblock = 'CMakeMake'
+version = '3.13.2'
+
+homepage = 'https://embree.github.io'
+description = """High Performance Ray Tracing Kernels"""
+
+
+source_urls = ['https://github.com/embree/embree/archive/refs/tags']
+sources = ['v%(version)s.tar.gz']
+checksums = ['dcda827e5b7a606c29d00c1339f1ef00f7fa6867346bc46a2318e8f0a601c6f9']
+
+
+toolchain = {
+    'name': 'foss',
+    'version': '2022b'
+}
+
+toolchainopts = {'pic': True}
+
+
+dependencies = [
+    ('ispc', '1.15.0', '', SYSTEM),
+    ('tbb', '2020.3'),
```

Comment:
I haven't tested up to which newer TBB version is compatible with this Embree version. When Intel released the OneAPI toolbox I haven't checked how much does the API change, i.e., if I can simply switch to the new version or provide patches... Additionally the software I am working on uses this version with Embree and not the newer one. 

tl;dr sadly not possible.

Reply:
for getting an easyconfig accepted in the EB repo, we have a rule saying that for each toolchain generation there should only be 1 version of a software used as a dependency for other software. in the case of `2022b`, the latest generation, we should really try to use the lastest `tbb` version if possible.

can you test this? the chances are high that it is backwards compatible. if not, i suggest using the `2021b` generation instead.

Reply:
I understand, then I will close the PR. The software is frozen in terms of versions, and I do not have time to see if upgrading to latest Embree/Tbb (now I see Embree is >4.0.0) or to the version that works with the 2022b toolchain is compatible with all the software packages in SALOME. 

The point of this PR was to have the dependencies of a program I am working on (I have a EC but for a different repository) and then to gradually include all the EasyConfigs of the SALOME framework, so that it is not installed as a Binary EasyConfig. I also intend to always use the newest toolchain but given the policy this will just make a big mess.